libfwupd/fwupd-client-sync.c

```diff
@@ -515,10 +515,11 @@ fwupd_client_get_details(FwupdClient *self,
 	}
 	return g_steal_pointer(&helper->array);
 #else
-	g_set_error_literal(error,
-			    FWUPD_ERROR,
-			    FWUPD_ERROR_NOT_SUPPORTED,
-			    "Not supported as <glib-unix.h> is unavailable");
+	g_set_error_literal(
+	    error,
+	    FWUPD_ERROR,
+	    FWUPD_ERROR_NOT_SUPPORTED,
+	    "fwupd_client_get_details not supported as <glib-unix.h> is unavailable");
```

Comment:
I think these should be nicer to the eye, e.g. "Getting details not supported as <glib-unix.h> is unavailable"